rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -50,7 +50,7 @@ class EmbeddingIntentClassifier(Component):
 
     provides = ["intent", "intent_ranking"]
 
-    requires = [MESSAGE_VECTOR_FEATURE_NAMES[MESSAGE_TEXT_ATTRIBUTE]]
+    requires = [MESSAGE_VECTOR_SPARSE_FEATURE_NAMES[MESSAGE_TEXT_ATTRIBUTE]]
```

Comment:
what do you mean? why it is impossible to use spacy vectors?

Reply:
Impossible was the wrong word. I rather meant even if you have `SpacyFeaturizer` in the pipeline its dense features will not be automatically loaded, because in the code you only load sparse features

Reply:
yes, will change also in one of the next PRs, same as above, just changed it here in order to pass the tests